### PR TITLE
fix: error check in the ensureFile func

### PR DESCRIPTION
**Pull request by testwill**

### CODE REVIEW

**Review by fahedouch — APPROVED** [review]
Thanks

File: pkg/dnsutil/hostsstore/hostsstore.go [hunk 1]
```diff
@@ -56,7 +56,7 @@ func ensureFile(path string) error {
 		return err
 	}
 	f, err := os.OpenFile(path, os.O_CREATE, 0644)
```
Comment (AkihiroSuda) [inline pkg/dnsutil/hostsstore/hostsstore.go:58]: Why not use `os.WriteFile`?

Comment (fahedouch) [reply pkg/dnsutil/hostsstore/hostsstore.go]: no need to write here, IMO `os.OpenFile` was fine

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**